static/js/services/rooms.js

```diff
@@ -135,6 +135,7 @@ define([
 				delete room.Credentials;
 			}
 			currentRoom = room;
+			$rootScope.roomType = room ? room.Type : null;
```

Comment:
This should not be stored at the rootScope (at least not here). I suggest to handle this where the room.updated event gets used (and hopefully at some scope sufficiently up in the hierachy which already stores runtime stuff).